src/operator/slice_channel.cc

```diff
@@ -29,33 +29,44 @@ DMLC_REGISTER_PARAMETER(SliceChannelParam);
 MXNET_REGISTER_OP_PROPERTY(SliceChannel, SliceChannelProp)
 .describe(R"code(Splits an array along a particular axis into multiple sub-arrays.
 
-Assume the input array has shape ``(d_0, ..., d_n)`` and we slice it into *m*
-(``num_outputs=m``) subarrays along axis *k*, then we will obtain a list of *m*
-arrays with each of which has shape ``(d_0, ..., d_k/m, ..., d_n)``.
+.. note:: ``SliceChannel`` is depreacted. Use ``split`` instead.
 
-For example::
+**Note** that `num_outputs` should evenly divide the length of the axis along
+which the array will be split.
```

Comment:
Suggest rewording the last part of the sentence as:

"... the axis along which to split the array."